mods/cnc/weapons/other.yaml

```diff
@@ -23,85 +23,29 @@ Flamethrower:
 	Warhead@3Eff: CreateEffect
 		Explosions: small_napalm
 		ImpactSounds: flamer2.aud
+		TargetSearchRadius: 0
+
+Flamethrower:
+	Inherits: ^FlameWeapon
 
 BigFlamer:
-	ValidTargets: Ground, Trees
+	Inherits: ^FlameWeapon
 	ReloadDelay: 50
 	Range: 3c512
-	InvalidTargets: Wall
-	Report: flamer2.aud
 	Projectile: Bullet
 		Speed: 341
 	Burst: 2
 	BurstDelay: 25
 	Warhead@1Dam: SpreadDamage
 		Spread: 400
 		Damage: 75
-		InvalidTargets: Wall
-		ValidTargets: Ground, Trees
 		Versus:
-			None: 100
-			Wood: 100
-			Light: 100
 			Heavy: 25
-		DamageTypes: Prone50Percent, TriggerProne, FireDeath, Incendiary
-	Warhead@2Smu: LeaveSmudge
-		SmudgeType: Scorch
-		InvalidTargets: Vehicle, Structure, Wall
 	Warhead@3Eff: CreateEffect
 		Explosions: med_napalm
-		ImpactSounds: flamer2.aud
-
-Chemspray:
-	ReloadDelay: 65
-	Range: 3c0
-	Report: flamer2.aud
-	Projectile: Bullet
-		Speed: 1c682
-	Warhead@1Dam: SpreadDamage
-		Spread: 256
-		Damage: 80
-		Versus:
-			None: 100
-			Wood: 35
-			Light: 75
-			Heavy: 50
-		DamageTypes: Prone50Percent, TriggerProne, TiberiumDeath
-	Warhead@2Smu: LeaveSmudge
-		SmudgeType: Scorch
-		InvalidTargets: Vehicle, Structure, Wall, Husk, Trees, Creep
-	Warhead@3Eff: CreateEffect
-		Explosions: chemball
-		ImpactSounds: xplos.aud
-
-Grenade:
-	ReloadDelay: 50
-	Range: 4c0
-	Report: toss1.aud
-	Projectile: Bullet
-		Speed: 140
-		Blockable: false
-		LaunchAngle: 62
-		Inaccuracy: 213
-		Image: BOMB
-	Warhead@1Dam: SpreadDamage
-		Spread: 341
-		Damage: 50
-		Versus:
-			None: 100
-			Wood: 50
-			Light: 80
-			Heavy: 35
-		DamageTypes: Prone50Percent, TriggerProne, SmallExplosionDeath
-	Warhead@2Smu: LeaveSmudge
-		SmudgeType: Crater
-		InvalidTargets: Vehicle, Structure, Wall, Husk, Trees, Creep
-	Warhead@3Eff: CreateEffect
-		Explosions: small_poof
-		ImpactSounds: xplos.aud
 
 Napalm:
-	ValidTargets: Ground, Water, Trees
+	Inherits: ^FlameWeapon
```

Comment:
This adds `Speed: 1c682` to the projectile, which will affect the airstrike balance.